analytics.js

```diff
@@ -497,6 +498,11 @@ class Analytics {
     if (!properties) {
       properties = {};
     }
+    Object.keys(properties).forEach((property) => {
+      if (reservedPropKeywords.indexOf(property) >= 0) {
+        console.error(`Warning! : Reserved keyword used --> ${property} with page call for name --> ${name}`);
```

Comment:
Should we use `logger` here and use a `warning` log level instead of `console.error`?